htdocs/web_portal/controllers/user/register.php

```diff
@@ -56,10 +56,20 @@ function draw() {
     die();
     }
 
+    //Extract users email from oidc claims
+    $authDetails = $_SERVER['OIDC_CLAIM_external_authn'];
```

Comment:
Not sure how it would work but it feels like the global OIDC_CLAIM_* values are crying out for an accessor class.